site/en/tutorials/structured_data/imbalanced_data.ipynb

```diff
@@ -968,7 +968,7 @@
         "id": "Y5twGRLfNwmO"
       },
       "source": [
-        "### Plot the ROC\r\n",
+        "### Plot the PR Curves\r\n",
```

Comment:
Suggestion:

- Since this section about _plotting the AUPRC_ follows the one about _plotting the ROC_, maybe we should fix this step description as follows to maintain consistency:

```suggestion
        "### Plot the AUPRC\r\n",
```

(emphasis on the Area Under the Curve of the Precision-Recall Curve to be more specific)

Reply:
yea @8bitmp3 that looks good. Either looks fine to me.